plugins/operators/__init__.py

```diff
@@ -669,28 +641,33 @@ def _clear_sample_field_inputs(ctx, inputs):
     schema.pop("id", None)
     schema.pop("filepath", None)
 
-    field_keys = list(schema.keys())
-    field_selector = types.AutocompleteView()
-    for key in field_keys:
-        field_selector.add_choice(key, label=key)
+    field_names = ctx.params.get("field_names", None) or []
 
-    field_prop = inputs.enum(
-        "field_name",
-        field_selector.values(),
+    field_choices = types.AutocompleteView(multiple=True)
```

Comment:
Suggestion: add `allow_duplicates=False` which prevents adding a given value multiple times. Also `multiple` is not needed since the setting is inferred from the schema being a `list`.

```
types.AutocompleteView(allow_duplicates=False)
```